hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/upgrade/FlinkUpgradeDowngrade.java

```diff
@@ -43,7 +43,9 @@ public void run(HoodieTableMetaClient metaClient, HoodieTableVersion toVersion,
 
   @Override
   protected void upgrade(HoodieTableVersion fromVersion, HoodieTableVersion toVersion, String instantTime) {
-    if (fromVersion == HoodieTableVersion.ZERO && toVersion == HoodieTableVersion.ONE) {
+    if (fromVersion == HoodieTableVersion.ONE && toVersion == HoodieTableVersion.TWO) {
+      // TODO:
```

Comment:
just throw exception if not implemented yet?

Reply:
+1